worlds/osrs/docs/setup_en.md

```diff
@@ -0,0 +1,53 @@
+# Setup Guide for Old School Runescape
+
+## Required Software
+
+- [RuneLite](https://runelite.net/)
+- If the account being used has been migrated to a Jagex Account, the [Jagex Launcher](https://www.jagex.com/en-GB/launcher)
+will also be necessary to run RuneLite
+
+## Configuring your YAML file
+
+### What is a YAML file and why do I need one?
+
+Your YAML file contains a set of configuration options which provide the generator with information about how it should
+generate your game. Each player of a multiworld will provide their own YAML file. This setup allows each player to enjoy
+an experience customized for their taste, and different players in the same multiworld can all have different options.
+
+### Where do I get a YAML file?
+
+You can customize your settings by visiting the 
+[Old School Runescape Player Settings Page](/games/Old%20School%20Runescape/player-settings)
```

Comment:
There is a crusade right now against "settings", it's gradually getting changed to "options" everywhere. You should probably follow suit